ifrs17-template/Report/ActualsUseCaseDataImport.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
@dcolleoni Where should this file go?
It imports the data.
Also, should the Reports on this use case be in this folder?

Reply:
Went for the dedicated folder approach